src/buffers/RecordingBuffer.cpp

```diff
@@ -79,11 +84,15 @@ int RecordingBuffer::Read(byte *buffer, size_t length)
   if (dataRead == 0 && m_isLive)
   {
     kodi::Log(ADDON_LOG_DEBUG, "%s:%d: %lld %lld", __FUNCTION__, __LINE__, m_inputHandle.GetLength() , m_inputHandle.GetPosition());
-    int64_t position = m_inputHandle.GetPosition();
-    Buffer::Close();
-    Buffer::Open(m_recordingURL, ADDON_READ_NO_CACHE);
-    Seek(position, 0);
-    dataRead = (int) m_inputHandle.Read(buffer, length);
+    const int64_t position = m_inputHandle.GetPosition();
+    const time_t startTime = time(nullptr);
```

Comment:
Just. a comment. The const is not really required here as both variables are integral types. You can still use it to indicate this variable should be changed but generally it gets used for complex types. Just a comment though and entirely up to you.

Reply:
Ok I didn't know that I thought Kodi want const for checks on function calls.  I will keep this in for now